libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/IncidentAlert.kt

```diff
@@ -0,0 +1,251 @@
+        return "IncidentAlert(" +
+            "incidentType=$incidentType" +
+            "incidentId=$incidentId" +
+            "incidentId=$incidentId" +
+            "incidentCongestion=$incidentCongestion" +
+            "isClosed=$isClosed" +
+            "creationTime=$creationTime" +
+            "startTime=$startTime" +
+            "endTime=$endTime" +
+            "description=$description" +
+            "subType=$subType" +
+            "subTypeDescription=$subTypeDescription" +
+            "alertcCodes=$alertcCodes" +
+            "), ${super.toString()}"
+    }
+
+    /**
+     * Use to create a new instance.
+     *
+     * @see IncidentAlert
+     */
+    class Builder(
+        private val coordinate: Point,
+        private val distance: Double
+    ) {
+        private var alertGeometry: RouteAlertGeometry? = null
+        private var incidentType: Int = IncidentAlertType.INCIDENT_UNKNOWN
+        private var incidentId: String? = null
+        @IncidentAlertImpact.Impact private var impact: String? = null
+        private var incidentCongestion: IncidentCongestion? = null
+        private var isClosed: Boolean = false
+        private var creationTime: Date? = null
+        private var startTime: Date? = null
+        private var endTime: Date? = null
+        private var description: String? = null
+        private var subType: String? = null
+        private var subTypeDescription: String? = null
+        private var alertcCodes: List<Int>? = null
+
+        /**
+         * Add optional geometry if the alert has a length.
+         */
+        fun alertGeometry(alertGeometry: RouteAlertGeometry?): Builder = apply {
+            this.alertGeometry = alertGeometry
+        }
+
+        /**
+         * Unique identifier for incident. Must be **non-null**.
+         */
+        fun incidentId(incidentId: String): Builder = apply {
+            this.incidentId = incidentId
+        }
+
+        /**
+         * One of [IncidentAlertType].
+         */
+        fun incidentType(incidentType: Int): Builder = apply {
+            this.incidentType = incidentType
+        }
+
+        /**
+         * Severity level of incident.
+         */
+        fun impact(@IncidentAlertImpact.Impact impact: String?): Builder = apply {
+            this.impact = impact
+        }
+
+        /**
+         * Quantitative descriptor of congestion.
+         */
+        fun incidentCongestion(incidentCongestion: IncidentCongestion?): Builder = apply {
+            this.incidentCongestion = incidentCongestion
+        }
+
+        /**
+         * **True** if road is closed and no possibility to pass through there. **False**
+         * otherwise.
+         */
+        fun isClosed(isClosed: Boolean): Builder = apply {
+            this.isClosed = isClosed
+        }
+
+        /**
+         * Time the incident was created/updated in ISO8601 format. Incident can be
+         * created/updated before the incident.
```

Comment:
```suggestion
         * Time the incident was created/updated in ISO8601 format. The incident alert can be
         * created/updated at a different time than its occurrence.
```

Is this suggestion true?

Reply:
/cc @danpaz for a sanity check

Reply:
cc @PureW @kevinkreiser @danpat @oini

Reply:
Yes @LukasPaczos this suggestion is correct.